# HG changeset patch
# User Rik <rik@octave.org>
# Date 1343275101 25200
#      Wed Jul 25 20:58:21 2012 -0700
# Node ID dd4ad69e4ab9ef46d2ecdfc991493a5f38af952c
# Parent  005cb78e1dd127981b0090d10eacdf3763ee0978
maint: Fix a few Octave coding convention violations.

* gzip.m, findfigs.m, hdl2struct.m: Use parentheses around if block test.

* graphics.cc, ov-class.cc: Use space after if and before first parenthesis

diff --git a/scripts/miscellaneous/gzip.m b/scripts/miscellaneous/gzip.m
--- a/scripts/miscellaneous/gzip.m
+++ b/scripts/miscellaneous/gzip.m
@@ -46,20 +46,20 @@ endfunction
 %! unwind_protect
 %!   filename = tmpnam;
 %!   dummy    = 1;
 %!   save (filename, "dummy");
 %!   dirname  = tmpnam;
 %!   mkdir (dirname);
 %!   entry = gzip (filename, dirname);
 %!   [path, basename, extension] = fileparts (filename);
-%!   if ! strcmp (entry, [dirname, filesep, basename, extension, ".gz"])
+%!   if (! strcmp (entry, [dirname, filesep, basename, extension, ".gz"]))
 %!     error ("gzipped file does not match expected name!");
 %!   endif
-%!   if ! exist (entry, "file")
+%!   if (! exist (entry, "file"))
 %!     error ("gzipped file cannot be found!");
 %!   endif
 %!   gunzip (entry);
 %!   if (system (sprintf ("diff %s %s%c%s%s", filename, dirname, filesep,
 %!                                            basename, extension)))
 %!     error ("unzipped file not equal to original file!");
 %!   end
 %! unwind_protect_cleanup
diff --git a/scripts/plot/findfigs.m b/scripts/plot/findfigs.m
--- a/scripts/plot/findfigs.m
+++ b/scripts/plot/findfigs.m
@@ -42,17 +42,17 @@ function findfigs ()
 
   ## give the monitor a margin so that the figure must not just
   ## marginally be on the monitor.
   margin = 30;
   screensize(1:2) += margin;
   screensize(3:4) -= margin;
 
   for i = 1:numel (figh)
-    if strcmp (get (figh(i), "visible"), "on")
+    if (strcmp (get (figh(i), "visible"), "on"))
 
       units = get (figh(i), "units");
       unwind_protect
         if (!strcmp (units, "pixels"))
           set (figh(i), "units", "pixels");
         endif
         pos = get (figh(i), "position");
         ## Test if (in order):
diff --git a/scripts/plot/hdl2struct.m b/scripts/plot/hdl2struct.m
--- a/scripts/plot/hdl2struct.m
+++ b/scripts/plot/hdl2struct.m
@@ -28,17 +28,17 @@
 
 function s = hdl2struct (h)
 
   if (nargin != 1 || !ishandle (h))
     print_usage ();
   endif
 
   hiddenh = get (0, "showhiddenhandles");
-  if strcmp (hiddenh, "on")
+  if (strcmp (hiddenh, "on"))
     set (0, "showhiddenhandles", "off");
   endif
 
   ## main object
   main = get (h);
   s.handle = h;
   s.type =  main.type;
   s.properties  = getprops (h);
@@ -77,17 +77,17 @@ function s = hdl2struct (h)
     nsp--;
   endwhile
 
   ## look for legends and colorbars among "main"'s brothers and add them
   ## to the children list
   if (strcmp (main.type, "axes"))
     par = main.parent;
     lg = findobj (par, "-depth", 1, "tag", "legend");
-    if !isempty (lg)
+    if (! isempty (lg))
       idx = arrayfun (@(x) get(x).userdata.handle(end) == h, lg);
       lg = lg(find (idx));
     endif
     nlg = length (lg);
     if (nlg == 1)
       ii++;
       s.children(ii) = hdl2struct (lg);
     elseif (nlg > 1)
@@ -145,17 +145,17 @@ function prpstr = getprops (h)
             "currentpoint", "extent"};
 
   nflds = length (fields);
   ii = 0;
   while nflds
     prop = fields{nflds};
     val = obj.(fields{nflds});
     ii++;
-    if !any (strcmp (prop, forbid))
+    if (! any (strcmp (prop, forbid)))
       prpstr.(prop) = val;
     endif
     nflds--;
   endwhile
 
   ## hidden properties
   hidden = {"autopos_tag", "looseinset"};
   for ii = 1:numel (hidden)
diff --git a/src/graphics.cc b/src/graphics.cc
--- a/src/graphics.cc
+++ b/src/graphics.cc
@@ -6828,19 +6828,19 @@ axes::properties::translate_view (double
 
 void
 axes::properties::rotate_view (double delta_el, double delta_az)
 {
   Matrix v = get_view ().matrix_value ();
 
   v(1) += delta_el;
 
-  if(v(1) > 90)
+  if (v(1) > 90)
     v(1) = 90;
-  if(v(1) < -90)
+  if (v(1) < -90)
     v(1) = -90;
 
   v(0) = fmod (v(0) - delta_az + 720,360);
 
   set_view (v);
   update_transform ();
 }
 
diff --git a/src/ov-class.cc b/src/ov-class.cc
--- a/src/ov-class.cc
+++ b/src/ov-class.cc
@@ -1272,17 +1272,17 @@ octave_class::reconstruct_parents (void)
   // First, check to see if there might be an issue with inheritance.
   for (octave_map::const_iterator p = map.begin (); p != map.end (); p++)
     {
       std::string  key = map.key (p);
       Cell         val = map.contents (p);
       if ( val(0).is_object () )
         {
           dbgstr = "blork";
-          if( key == val(0).class_name () )
+          if ( key == val(0).class_name () )
             {
               might_have_inheritance = true;
               dbgstr = "cork";
               break;
             }
         }
     }
 
